src/anemoi/graphs/processors/post_process.py

```diff
@@ -36,20 +36,21 @@ def __init__(
     ) -> None:
         self.nodes_name = nodes_name
         self.save_mask_indices_to_attr = save_mask_indices_to_attr
+        self.mask: torch.Tensor = None
 
-    def removing_nodes(self, graph: HeteroData, mask: torch.Tensor) -> HeteroData:
+    def removing_nodes(self, graph: HeteroData) -> HeteroData:
         """Remove nodes based on the mask passed."""
         for attr_name in graph[self.nodes_name].node_attrs():
-            graph[self.nodes_name][attr_name] = graph[self.nodes_name][attr_name][mask]
+            graph[self.nodes_name][attr_name] = graph[self.nodes_name][attr_name][self.mask]
```

Comment:
If one chooses the hidden mesh to post process on then, even though the mask is all true, when the `attr_name` loop hits `_node_ordering` the following error is generated
```
  File "/etc/ecmwf/nfs/dh2_home_b/bel0679/ai_stuff/anemoi_code/anemoi-graphs_pp/src/anemoi/graphs/processors/post_process.py", line 45, in removing_nodes
    graph[self.nodes_name][attr_name] = graph[self.nodes_name][attr_name][mask]
TypeError: only integer tensors of a single element can be converted to an index
```
Although applying the post processor to the hidden mesh is (currently) maybe not the intended use, it would be good if the code could handle it.